### DISCUSSION TITLE
文章置顶

### DISCUSSION THREAD

**Discussion by 2473o** [Tokinx/Wing#98] [讨论]
使用笔记模块后，无法使用置顶文章，如图
![image](https://user-images.githubusercontent.com/86139845/210940305-05f89263-9934-43a9-8b03-b66a1c4b31aa.png)
![image](https://user-images.githubusercontent.com/86139845/210940387-07e1686b-5b44-4f9c-8b29-0760386aa5c3.png)

**Comment by moxiaonai** [comment]
> 使用笔记模块后，无法使用置顶文章，如图 ![image](https://user-images.githubusercontent.com/86139845/210940305-05f89263-9934-43a9-8b03-b66a1c4b31aa.png) ![image](https://user-images.githubusercontent.com/86139845/210940387-07e1686b-5b44-4f9c-8b29-0760386aa5c3.png)

@2473o 置顶我这边实现了，给作者提了pr，希望可以被合入，可以关注https://github.com/Tokinx/Wing/pull/103
